x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ssl/SSLService.java

```diff
@@ -656,6 +647,35 @@ private void validateServerConfiguration(String prefix) {
         }
     }
 
+    private void maybeValidateRemoteClusterServerConfiguration() {
```

Comment:
I can change it to `== false`. 
Some explanation on this style: I personally prefer this style because I feel it reads better. Noramlly a negate check is `!condition` and I read it as "not condition". Since we don't use `!` for negate, I find the style `false == condition` reads the same way for me. As soon as I see the leading `false`, I am reading it as "not condition". But if the `false` is at the end, I find it is less readable because I don't know it is a negate till the end.

Reply:
Pushed https://github.com/elastic/elasticsearch/pull/93334/commits/29185f39e8364e63d7e83a620397a58f1b3cf302